Command/Phinx/CommandBuilder.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace PCF\MagentoMigration\Command\Phinx;
+
+use mikehaertl\shellcommand\Command;
+use PCF\MagentoMigration\Api\PathLocatorInterface;
+use PCF\MagentoMigration\Api\ConfigBuilderInterface;
+
+class CommandBuilder
+{
+
+    /** @var PathLocatorInterface */
+    protected $pathLocator;
+
+    /** @var ConfigBuilderInterface */
+    protected $configBuilder;
+
+    /** @var Command */
+    protected $command;
+
+    /**
+     * Phinx constructor.
+     * @param PathLocatorInterface $pathLocator
+     * @param ConfigBuilderInterface $configBuilder
+     * @param Command $command
+     */
+    public function __construct(
+        PathLocatorInterface $pathLocator,
+        ConfigBuilderInterface $configBuilder,
+        Command $command
+    ) {
+        $this->pathLocator = $pathLocator;
+        $this->configBuilder = $configBuilder;
+        $this->command = $command;
+    }
+
+
+    public function getExecCommand($method = '')
```

Comment:
php doc and returning type.

Reply:
done. also repaired tests for that method